src/components/filters/FilterList/index.jsx

```diff
@@ -6,8 +6,11 @@ import './FilterList.css';
 class FilterList extends React.Component {
   constructor(props) {
     super(props);
-    const initialFilterStatus = props.sections
+    let initialFilterStatus = props.sections
       .map(() => ({}));
+    if (props.filterStatusFromParent && Object.keys(props.filterStatusFromParent).length > 0) {
+      initialFilterStatus = props.filterStatusFromParent.map(x => Object.assign({}, x));
```

Comment:
I think you can use `initialFilterStatus = [...props.filterStatusFromParent]` instead